frontend/api/authors.js

```diff
@@ -1,67 +1,65 @@
-import { gql, useQuery, useMutation } from "@apollo/client";
-
-export const useGetAuthors = () => {
-  const query = gql`
-    query getAuthors {
-      getAuthors {
-        id
-        firstName
-        lastName
-      }
+import { gql, useQuery, useMutation } from '@apollo/client';
+export const allAuthorsQuery = gql`
+  query getAuthors {
+    getAuthors {
+      id
+      firstName
+      lastName
     }
-  `;
-  const { loading, error, data } = useQuery(query);
+  }
+`;
+export const useGetAuthors = () => {
+  const { loading, error, data } = useQuery(allAuthorsQuery);
   return {
     authorsLoading: loading,
     authorsError: error,
-    authors: data?.getAuthors || [],
+    authors: data?.getAuthors || []
   };
 };
 
-export const useGetAuthor = (id) => {
-  const query = gql`
-    query getAuthor($id: ID!) {
-      getAuthor(id: $id) {
+export const anAuthorQuery = gql`
```

Comment:
Curious why we export this, typically just need these queries in only this file, no?

Reply:
I did the export to support the test mocks as the request required the query:

const getBooksMOCK = {
      request: {
        query: allBooksQuery,
        variables: {}
      },....